terraform/a2ha-terraform/deployment-makefile/Makefile

```diff
@@ -0,0 +1,86 @@
+# Global defaults
+TERRAFORM_PATH = $(shell pwd)/terraform
```

Comment:
can we have only the relevant content in the make file. as we are maintaining 2 make file.

Reply:
I have already removed irrrelavent content from the Makefile before creating a PR. you can compare both Makefile.